.circleci/config.yml

```
@@ -47,18 +47,19 @@ jobs:
           fi
 
 
-  e2eTestChartInstall:
+  e2eTestChartBasic:
```

```suggestion
  e2eTestAppBasic:
```

I think this naming is better. As its the app controller that is being tested.